indico/modules/events/papers/notifications.py

```diff
@@ -57,20 +57,21 @@ def notify_paper_review_submission(review):
 
 def notify_paper_judgment(paper, reset=False):
     event = paper.event_new
-    receiver = paper.last_revision.submitter
-    template = None
+    receivers = ([x for x in paper.contribution.submitters if x.email] if paper.last_revision.submitter.is_janitor
```

Comment:
why not use authors instead of submitters? material submission rights aren't taken into account for anything else in paper reviewing (iirc we discussed this a few times already ;)) so i don't see why they should be used here